jupyter_forward/core.py

```diff
@@ -229,13 +230,14 @@ def _conda_activate_cmd(self):
         check_jupyter_status = 'which jupyter'
         activate_cmds = ['source activate', 'conda activate']
 
-        # Check for mamba availability and prioritize it if found
-        try:
-            mamba_check = self.run_command('which mamba', warn=False, echo=False, exit=False)
-            if not mamba_check.failed:
-                activate_cmds = ['mamba activate']
-        except Exception as e:
-            console.print(f'[bold yellow]:warning: Mamba check failed: {e}')
+        # Check for micrmamba, then mamba availability and prioritize
+        # which ever is found first
+        if shutil.which('micromamba') is not None:
+            activate_cmds = ['micromamba activate']
+        elif shutil.which('mamba') is not None:
+            activate_cmds = ['mamba activate']
```

Comment:
since the command is meant to be executed on a "remote" machine, we should use `self.run_command('which micromamba', warn=False, echo=False, exit=False)` here instead of `shutil` which assumes we are executing this command on the same machine `jupyter-forward` is invoked from

Reply:
Ah, I see. I wanted to avoid nested exceptions.